docs/starters.yml

```diff
@@ -3639,6 +3639,7 @@
     - Ready made Components
     - Responsive Design
     - Syntax Highlight in Code.
+- url: https://gastby-starter-organization.netlify.com/
```

Comment:
This should be removed?

Reply:
Nope

Reply:
This is unrelated to your entry, correct?